spyder/widgets/sourcecode/codeeditor.py

```diff
@@ -2425,6 +2425,7 @@ def __get_current_color(self, cursor=None):
 
     def in_comment_or_string(self, cursor=None):
         """Is the cursor inside or next to a comment or string?"""
+        raise Exception

```

Comment:
This is making our tests to fail. It seems an artifact of the rebase, so please remove it.